src/citrine/resources/data_concepts.py

```diff
@@ -79,56 +85,68 @@ def build(cls, data: dict, session: Session = None):
             An object corresponding to a data concepts resource.
 
         """
+        data_copy = deepcopy(data)
+        # Extract the values that are only for the client.
+        # They will be attached to the deserialized object later.
+        client_only_dict = dict()
+        for client_key in cls._client_keys:
+            val = DataConcepts._pop_field(data_copy, client_key)
+            client_only_dict.__setitem__(client_key, val)
```

Comment:
Is this distinct from
```suggestion
            client_only_dict[client_key] = val
```
?